other.js

```diff
@@ -0,0 +1,3 @@
+// js file
+
```

Comment:
Should we add more here?

 - item 1
 - item 2

[Reference spec](https://tc39.github.io/proposal-dynamic-import/)

Reply:
Yes we should definitely add X here.